install-validator/pkg/dir/install.go

```diff
@@ -0,0 +1,42 @@
+package dir
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	kubewrapper "github.com/vmware-tanzu/graph-framework-for-microservices/install-validator/pkg/k8s-utils"
+	v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"sigs.k8s.io/yaml"
+)
+
+func InstallDir(dir string, c kubewrapper.ClientInt) error {
+	err := filepath.Walk(dir, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return nil
```

Comment:
why are we completely ignoring error here? I Understand not to error when no files existing in the directory